Maddori.Apple/Maddori.Apple/Screen/Home/Home/HomeViewController.swift

```diff
@@ -91,8 +94,15 @@ final class HomeViewController: BaseViewController {
         }
         labelButton.subText = TextLiteral.mainViewControllerPlanLabelButtonSubText
         labelButton.subButtonText = TextLiteral.mainViewControllerPlanLabelButtonSubButtonText
+        labelButton.tag = 100
         return labelButton
     }()
+    private lazy var planLableButtonBackgroundView: UIView = {
```

Comment:
p3;

```suggestion
    private let planLableButtonBackgroundView: UIView = {
```
`let`으로 해도 괜찮을 것 같네요 !

Reply:
[c14ce7b](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/142/commits/c14ce7b23082229bcf7cb2ae6e39facb96512b63)
옙!